packages/react/src/hooks/useMediaTrackBySourceOrName.ts

```diff
@@ -16,7 +16,7 @@ export function useMediaTrackBySourceOrName(
   const [publication, setPublication] = React.useState(getTrackByIdentifier(observerOptions));
 
   const [isMuted, setMuted] = React.useState(publication?.isMuted);
-  const [isSubscribed, setSubscribed] = React.useState(publication?.isSubscribed);
+  const [isSubscribed, setSubscribed] = React.useState(publication?.isSubscribed ?? false);
```

Comment:
I guess, this changes the type of `isSubscribed` from boolean to `boolean | undefined`. 
can we avoid/revert that?